### PR TITLE
Update phonecalloriginmanager_issupported.md

### PR DESCRIPTION
**Pull request by wilsonNg007**
Update IsSupported behavior based on the updated design change.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@wilsonNg007 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: windows.applicationmodel.calls.provider/phonecalloriginmanager_issupported.md [hunk 1]
```diff
@@ -11,12 +11,13 @@ public bool IsSupported { get; }
...
 
 ## -description
-Returns true if PhoneCallOriginManager APIs are supported and false if they are not supported.
+Returns true if PhoneCallOriginManager APIs are supported and false if they are not supported. See remarks for exception. 
 
 ## -property-value
 The bool representing whether PhoneCallOriginManager APIs are supported or not.
 
 ## -remarks
+Exception: SetCallOrigin API is supported and not affected by this API
```
Comment (aschoorl) [inline windows.applicationmodel.calls.provider/phonecalloriginmanager_issupported.md:20]: Are you updating the documentation for SetCallOrigin as well? It still talks about China waivers which doesn't seem relevant

Comment (wilsonNg007) [reply windows.applicationmodel.calls.provider/phonecalloriginmanager_issupported.md:20]: Sure.  I can update the other ones.

Comment (wilsonNg007) [reply windows.applicationmodel.calls.provider/phonecalloriginmanager_issupported.md:20]: Here it's - https://github.com/MicrosoftDocs/winrt-api/pull/1690

**Comment by MatchaMatch** [comment]
Thanks for your contribution! I've merged your changes and they should be live soon.

**Pull request merged**